examples/text_classification/bert/bert_handler.py

```diff
@@ -13,7 +13,7 @@
 class Bertseqclassifier(object):
     """
     Bertseqclassifier handler class. This handler takes a piece of text
-    and returns the classification prediction which here is 4 classes in ag_news as presented in
+    and returns the classification prediction which here is 4 classes in ag_news dataset as presented in
```

Comment:
@HamidShojanazeri Please align the comments and the index_to_name.json -- to match. At present index file only have 2 categories and comments mention 4 classes